hugo.yaml

```diff
@@ -1,4 +1,4 @@
-baseURL: https://docs.wandb.ai
```

Comment:
Please move this to a separate PR for review.

Reply:
This was in this PR to work around the broken image in the PR preview. If I were not removing the image based on your feedback, it would make sense to keep it together.

Reply:
Moved to https://github.com/wandb/docs/pull/1244